zutil.h

```diff
@@ -76,6 +76,7 @@ extern z_const char * const PREFIX(z_errmsg)[10]; /* indexed by 2-zlib_error */
 #define PRESET_DICT 0x20 /* preset dictionary flag in zlib header */
 
 #define ADLER32_INITIAL_VALUE 1 /* initial adler-32 hash value */
+#define CRC32_INITIAL_VALUE 0 /* initial crc-32 hash value */
```

Comment:
Comments should start at same column if there is no empty line in between...

Reply:
Fixed.